### ISSUE TITLE
Macbook 12 second gen status. 

### ISSUE THREAD

**Issue by giulioungaretti** [Floris497/mac-pixel-clock-patch-V2#34]
## Before  patch:
- Setup 1:
  - macbook, apple media adapter (firmware Version:2.33) , hdmi --> asus PA328Q 
- Status:
   - no output 
- Setup 2:
  - macbook, dell wd 15  dock, display port  --> asus PA328Q 
- Status:
  - no output 

Will update soon with after patch test.

**Comment by giulioungaretti** [comment]
## After patch:
- Setup 1:
  - macbook, apple media adapter (firmware Version:2.33) , hdmi --> asus PA328Q 
- Status:
  -  4k @30hz, no option to select 60hz 
- Setup 2:
  - macbook, dell wd 15  dock, display port  --> asus PA328Q 
- Status:
  -  4k @30hz, no option to select 60hz.
  -  the connection between macbook and display is wonky (sometimes it does not wake up)

**Comment by giulioungaretti** [comment]
#system 

![screen shot 2016-06-21 at 11 33 23](https://cloud.githubusercontent.com/assets/2694199/16224811/ff6907ea-37a3-11e6-950b-2cb2823ffa54.png)

**Comment by Floris497** [comment]
You don't get lower resolutions without the patch?

**Comment by giulioungaretti** [comment]
@Floris497  this is what I get :D! 
<img width="677" alt="screen shot 2016-06-21 at 11 45 12" src="https://cloud.githubusercontent.com/assets/2694199/16225179/a9623cca-37a5-11e6-8d0c-f459d066725d.png">

**Comment by giulioungaretti** [comment]
@Floris497  sorry, misread your question. No I don't even get signal out without the patch :D 
Maybe because of the last usb-c update from apple ?

**Comment by Floris497** [comment]
Maybe.. I have no idea what they updated.. But it seems weird to me that it doesn't give signal before the patch.. That's what i don't get.. What type of connection are you using to the display? Only HDMI2.0 and DisplayPort can do 4k@60Hz (USB-C monitors use native DisplayPort capabilities of USB-C port)

**Comment by giulioungaretti** [comment]
I am using a usb-c dock, that has display port out, display supports dp1.2.
And then the apple adapter with HDMI (don't know if it supports HDMI 2.0).

**Comment by Floris497** [comment]
The apple one is NOT HDMI2.0 (most probably HDMI1.4a) Why the DisplayPort dock does not output 60Hz.. I have no idea. Look if your monitor has MST SST settings (sometimes referred to as DP1.1/DP1.2) It can be the case.

**Comment by giulioungaretti** [comment]
@Floris497  yes, the monitor has a d1.2 and it's enabled. But maybe the dock is doing something weird.
I should try to do use a usb-c -- > display port cable!

**Comment by Floris497** [comment]
Use the google one.. Should work

**Comment by giulioungaretti** [comment]
@Floris497 thanks! Will try to do this!
